src/main/java/com/webonise/rbs/repository/BookingRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.webonise.rbs.repository;
+
+import com.webonise.rbs.entity.Booking;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface BookingRepository extends JpaRepository<Booking, Long> {
+}
```

Comment:
Add new line, do this change for other class also.
https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file